### DISCUSSION TITLE
Please help to group habits into categories and sub categories

### DISCUSSION THREAD

**Discussion by eternallearnings1** [iSoron/uhabits#2008] [Feature Requests]
Request is to move habits into categories and sub categories folders

**Comment by ferous531** [comment]
This is the most surprising "lack of fesuture" in the app. I have exercise habits, dietary habits, spiritual habits, and relational habits. It's strange not being able to organize habits into those categories

**Comment by ezio-melotti** [comment]
* See https://github.com/iSoron/uhabits/discussions/495
